remove line that breaks the portal (#200)
IBM/cp4i-deployment-samples: products/bash/apic/README.md

## Usage

> - PLEASE NOTE: The `configure-apic-v10.sh` is for Demos only and not recommended for use anywhere else. The script uses unsupported internal features that are NOT suitable for production usecases.

Reviewer: ```suggestion
- PLEASE NOTE: The `configure-apic-v10.sh` script uses unsupported internal features and is for demo purposes ONLY. It is NOT suitable for production use cases.
```
